packages/web-components/src/components/cbp-list/cbp-list.stories.tsx

```diff
@@ -312,4 +312,55 @@ export const IconLinkList = IconListTemplate.bind({});
             color: "var(--cbp-color-red-50)",
           },
         ]
-};
\ No newline at end of file
+};
+
+function generateDescriptionItems(items) {
+  const html = items.map(({ content, description, icon, color}) => {
+      return `
+              <dt>
+                <cbp-icon name='${icon}' color='${color}' size='1.25rem'></cbp-icon>
+                ${content}
+              </dt>
+              <dd>
+                ${description}
+              </dd>
+              `;      
+  });
+  return html.join('');
+}
+
+const DescriptionListTemplate = ({linkListItems, accessibilityText, context,}) => {
+  return ` 
+    <cbp-list
+      tag='dl' 
```

Comment:
The tag control is still showing up and showing `ul` is selected but it's not being used. Remove the control for this story or see if you can remove the other tags for this instance.